packages/gatsby-plugin-onesignal/src/gatsby-ssr.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```

Comment:
Did you consider adding this plugin lazy or inside the gastby-browser ?

Reply:
The script tags are being added to the final HTML after the `<body>` tag and loaded using the `async` flag, so I think it doesn't affect the page's performance, at least not in any of the Lighthouse metrics.

Adding this in the SSR stage seems cheaper to me since adding it to `gatsby-browser` would make it so that this code gets executed during the page load, right?